content/2019/Making-things-flow/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Making Things Flow"
+publishDate: 2019-12-01
```

Comment:
```suggestion
publishDate: 2020-01-01
```

Reply:
I've made this `2020-01-03` as the 1st has passed.